features/networking/multus.feature

```diff
@@ -1056,3 +1056,82 @@ Feature: Multus-CNI related scenarios
     Then the output should contain:
       | default via 22.2.2.254 dev net1  |
 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-22504
+  @admin
+  Scenario: The multus adminssion controller should be able to detect that the pod is using net-attach-def in other namespaces when the isolation is enabled
+    Given I create 2 new projects
+    # Create the net-attach-def via cluster admin
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["name"]      | macvlan-bridge-25657    |
+      | ["metadata"]["namespace"] | <%= project(-1).name %> |    
+    Then the step should succeed
+    Given I use the "<%= project(-2).name %>" project
+    # Create a pod in new project consuming net-attach-def from 1st project
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/1interface-macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | macvlan-bridge-25657 |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/pod\/(.*) created/)[1]` is stored in the :pod_name clipboard
+    #making sure the created pod complains about net-attach-def and hence stuck in ContainerCreating state
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :describe client command with:
+      | resource | pods               |
+      | name     | <%= cb.pod_name %> |
+    Then the step should succeed
+    And the output should contain:
+      | cannot find get a network-attachment-definition |
+      | ContainerCreating                               |
+    """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24492
+  @admin
+  Scenario: Create pod with Multus ipvlan CNI plugin	
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    And the default interface on nodes is stored in the :default_interface clipboard
+    #Storing default interface mac address for comparison later with pods macs
+    Given I use the "<%= cb.nodes[0].name %>" node
+    And I run commands on the host:
+      | ip addr show <%= cb.default_interface %> |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/\w{2}:\w{2}:\w{2}:\w{2}:\w{2}:\w{2}/)[0]` is stored in the :default_interface_mac clipboard
```

Comment:
could we use `\h`.   `\h` is a hexdigit since at least ruby 2.1.0

`/\h/ - A hexdigit character ([0-9a-fA-F])`

https://ruby-doc.org/core-2.6.5/doc/regexp_rdoc.html#label-Character+Classes

Reply:
Yeah, that make sense logically and good for readers as well. Thanks